critiquebrainz/frontend/static/styles/main.less

```diff
@@ -102,6 +102,11 @@ ul.sharing {
   margin-top: 4px;
 }
 
+// Search page
+.search-result-title {
+  width: 50%;
+}
```

Comment:
At some point I wonder if my brain is just trying to justify all the css it contains, but you could also do this:

```
#search-results thead tr th:first-child {
  width: 50%;
}
```

and add a single `id=`search-results"` to your `<table>` element to avoid repeating the class in each `if` block

Reply:
at the very least I like the idea of adding in an id to anchor the css a bit more to a specific page - I was thinking about doing that.